Lib9c/Action/CreateAvatar0.cs

```diff
@@ -199,7 +199,7 @@ public override IAccount Execute(IActionContext context)
             var consumableItemSheet = ctx.PreviousState.GetSheet<ConsumableItemSheet>();
             AddItemsForTest(
                 avatarState: avatarState,
-                random: ctx.Random,
+                random: ctx.GetRandom(),
```

Comment:
As `ctx.GetRandom()` always returns an `IRandom` of the same state, `ctx.GetRandom()` should be extracted first in order to preserve the behavior. 😶

Reply:
https://github.com/planetarium/lib9c/pull/2174/commits/630bc04bd209c7fb7d4ffeb5c1413e3f569ea67c amended